.editorconfig

```diff
@@ -124,71 +146,97 @@ csharp_style_inlined_variable_declaration = true:suggestion
 csharp_style_conditional_delegate_call = true:suggestion
 
+# Other features
+csharp_style_prefer_index_operator = true:none
+csharp_style_prefer_range_operator = true:none
+csharp_style_pattern_local_over_anonymous_function = true:none
+
 # Space preferences
-csharp_space_after_cast = false
-csharp_space_after_colon_in_inheritance_clause = true
-csharp_space_after_comma = true
-csharp_space_after_dot = false
+csharp_space_after_cast                                = false
+csharp_space_after_colon_in_inheritance_clause         = true
+csharp_space_after_comma                               = true
+csharp_space_after_dot                                 = false
 csharp_space_after_keywords_in_control_flow_statements = true
-csharp_space_after_semicolon_in_for_statement = true
-csharp_space_around_binary_operators = before_and_after
+csharp_space_after_semicolon_in_for_statement          = true
+csharp_space_around_binary_operators       = before_and_after
 csharp_space_around_declaration_statements = do_not_ignore
 csharp_space_before_colon_in_inheritance_clause = true
-csharp_space_before_comma = false
-csharp_space_before_dot = false
-csharp_space_before_open_square_brackets = false
-csharp_space_before_semicolon_in_for_statement = false
-csharp_space_between_empty_square_brackets = false
-csharp_space_between_method_call_empty_parameter_list_parentheses = false
-csharp_space_between_method_call_name_and_opening_parenthesis = false
-csharp_space_between_method_call_parameter_list_parentheses = false
+csharp_space_before_comma                       = false
+csharp_space_before_dot                         = false
+csharp_space_before_open_square_brackets        = false
+csharp_space_before_semicolon_in_for_statement  = false
+csharp_space_between_empty_square_brackets                               = false
+csharp_space_between_method_call_empty_parameter_list_parentheses        = false
+csharp_space_between_method_call_name_and_opening_parenthesis            = false
+csharp_space_between_method_call_parameter_list_parentheses              = false
 csharp_space_between_method_declaration_empty_parameter_list_parentheses = false
-csharp_space_between_method_declaration_name_and_open_parenthesis = false
-csharp_space_between_method_declaration_parameter_list_parentheses = false
-csharp_space_between_parentheses = false
-csharp_space_between_square_brackets = false
+csharp_space_between_method_declaration_name_and_open_parenthesis        = false
+csharp_space_between_method_declaration_parameter_list_parentheses       = false
+csharp_space_between_parentheses                                         = false
+csharp_space_between_square_brackets                                     = false
 
-# Wrapping Options
-csharp_preserve_single_line_blocks = true
-csharp_preserve_single_line_statements = false
-
-# C++ Files
-
-# IDE0052: Remove unread private members
-dotnet_diagnostic.IDE0052.severity = warning
+# Analyzers
```

Comment:
Everything under "Analyzers" includes the rules previously imposed by the ruleset files